tools/mo/unit_tests/mo/convert/meta_data_test.py

```diff
@@ -0,0 +1,150 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import tempfile
+from pathlib import Path
+
+from generator import generator
+from openvino.runtime import get_version as get_rt_version
+from openvino.runtime import serialize
+
+from openvino.tools.mo import convert
+from openvino.tools.mo.utils import import_extensions
+from openvino.tools.mo.utils.version import get_version
+from unit_tests.mo.unit_test_with_mocked_telemetry import UnitTestWithMockedTelemetry
+from utils import save_to_onnx
+
+
+@generator
+class MetaDataTest(UnitTestWithMockedTelemetry):
+    test_directory = os.path.dirname(os.path.realpath(__file__))
+
+    def test_meta_data(self):
+        def create_onnx_model():
+            #
+            #   Create ONNX model
+            #
+
+            import onnx
+            from onnx import helper
+            from onnx import TensorProto
+
+            shape = [1, 2, 3]
+
+            input = helper.make_tensor_value_info('input', TensorProto.FLOAT, shape)
+            output = helper.make_tensor_value_info('output', TensorProto.FLOAT, shape)
+
+            node_def = onnx.helper.make_node(
+                'Relu',
+                inputs=['input'],
+                outputs=['Relu_out'],
+            )
+            node_def2 = onnx.helper.make_node(
+                'Sigmoid',
+                inputs=['Relu_out'],
+                outputs=['output'],
+            )
+
+            # Create the graph (GraphProto)
+            graph_def = helper.make_graph(
+                [node_def, node_def2],
+                'test_model',
+                [input],
+                [output],
+            )
+
+            # Create the model (ModelProto)
+            onnx_net = helper.make_model(graph_def, producer_name='test_model')
+            return onnx_net
+
+        def ref_meta_data():
+            return {
+                'MO_version': get_version(),
+                'Runtime_version': get_rt_version(),
+                'legacy_path': "False",
+                'conversion_parameters': {
+                    'caffe_parser_path': Path("DIR"),
```

Comment:
Do we need it?

Reply:
This value will be in meta data only if it is set to non-default value by user.